rskj-core/src/main/java/co/rsk/peg/federation/Federation.java

```diff
@@ -93,6 +94,24 @@ public List<BtcECKey> getBtcPublicKeys() {
             .collect(Collectors.toList());
     }
 
+    public List<ECKey> getRskPublicKeys() {
```

Comment:
Were these methods added just for testing? Not sure that's a good idea if that's the case

Reply:
I agree, if that is the case consider making these methods package-private or use reflection utilities present in `TestUtils`: https://github.com/rsksmart/rskj/blob/e130ef722ca87eb881d4da435b30ec23f8fee15a/rskj-core/src/test/java/org/ethereum/TestUtils.java#L199